website/source/docs/providers/aws/r/autoscaling_notification.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_vpc"
+sidebar_current: "docs-aws-resource-autoscaling-notification"
+description: |-
+  Provides an AutoScaling Group with Notification support
+---
+
+# aws\_vpc
+
+Provides an AutoScaling Group with Notification support, via SNS Topics
```

Comment:
It's tricky to describe what this creates... they are resources in a since, but you can't access them directly... anyway, your point is valid. I'll expand on that.